app/helpers/template_helpers.py

```diff
@@ -42,6 +43,88 @@ def get_page_header_context(language, theme):
     return context.get(theme)
 
 
+def get_footer_context(language_code, static_content_urls, sign_out_url, theme):
+    default_context = {
+        "crest": True,
+        "newTabWarning": lazy_gettext("The following links open in a new tab"),
+        "copyrightDeclaration": {
+            "copyright": lazy_gettext(
+                "Crown copyright and database rights 2020 OS 100019153."
+            ),
+            "text": lazy_gettext("Use of address data is subject to the"),
+            "link": lazy_gettext("terms and conditions"),
+            "url": static_content_urls["terms_and_conditions"],
+            "target": "_blank",
+        },
+        "rows": [
+            {
+                "itemsList": [
+                    {
+                        "text": lazy_gettext("Help"),
+                        "url": static_content_urls["help"],
+                        "target": "_blank",
+                    },
+                    {
+                        "text": lazy_gettext("Contact us"),
+                        "url": static_content_urls["contact_us"],
+                        "target": "_blank",
+                    },
+                ]
+            },
+            {
+                "itemsList": [
+                    {
+                        "text": lazy_gettext("Cookies"),
+                        "url": static_content_urls["cookies"],
+                        "target": "_blank",
+                    },
+                    {
+                        "text": lazy_gettext("Accessibility statement"),
+                        "url": static_content_urls["accessibility_statement"],
+                        "target": "_blank",
+                    },
+                    {
+                        "text": lazy_gettext("Privacy and data protection"),
+                        "url": static_content_urls["privacy_and_data_protection"],
+                        "target": "_blank",
+                    },
+                    {
+                        "text": lazy_gettext("Terms and conditions"),
+                        "url": static_content_urls["terms_and_conditions"],
+                        "target": "_blank",
+                    },
+                ]
+            },
+        ],
+    }
+
+    if request.blueprint == "post_submission":
+        default_context["footerWarning"] = lazy_gettext(
+            "Make sure you <a href='{sign_out_url}'>leave this page</a> or close your browser if using a shared device"
+        ).format(sign_out_url=sign_out_url)
+
+    context = {
+        "default": {
+            **default_context,
+            "lang": language_code,
+        },
```

Comment:
The page header context also handles `social` and `northernireland` themes, so this should too for consistency.
We should be able to move `lang` into the default context, and the `census-nisra` dict merge below will overwrite it.